src/Badges.sol

```diff
@@ -2,8 +2,8 @@
 pragma solidity ^0.8.6;
 
 import {ERC4973} from "ERC4973/ERC4973.sol";
-import {Counters} from "@openzeppelin/contracts/utils/Counters.sol";
-import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+import {Counters} from "@openzeppelin-contracts/contracts/utils/Counters.sol";
```

Comment:
```suggestion
@openzeppelin/=lib/openzeppelin-contracts/
```

otherwise ChainClaim won't build bc of this issue https://github.com/foundry-rs/foundry/issues/1657